.buildkite/x-pack/pipeline.xpack.winlogbeat.yml

```diff
@@ -10,39 +12,136 @@ env:
   IMAGE_WIN_2022: "family/platform-ingest-beats-windows-2022"
   GCP_HI_PERF_MACHINE_TYPE: "c2d-highcpu-16"
   GCP_WIN_MACHINE_TYPE: "n2-standard-8"
-  BEATS_PROJECT_NAME: "x-pack/winlogbeat"
-
-  #Packaging
-  PACKAGING_PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
-  PACKAGING_ARM_PLATFORMS: "linux/arm64"
 
-  #Deps
-  ASDF_MAGE_VERSION: 1.15.0
+notify:
+  - github_commit_status:
+      context: "x-pack/winlogbeat"
```

Comment:
since we have notifications per step (already here for 2019: https://github.com/elastic/beats/pull/38851/files#diff-9976a7ded551339fec53006fed76cd77816d59e80ad9c5bdff77d87bc56b81a2R121) let's be consistent with other PRs, so remove this top level notification and have a dedicated notification per step.